### PR TITLE
add standaline install link, other cleanup

### PR DESCRIPTION
**Pull request by kbhawkey**
Introduced link to KFP standalone installation.
First stab at a bit of cleanup.
For further information, see issue #1623 .

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1791)

### CODE REVIEW

File: content/docs/pipelines/overview/pipelines-overview.md [hunk 1]
```diff
@@ -235,31 +236,29 @@ At a high level, the execution of a pipeline proceeds as follows:
 * **Pipeline Service**: You call the Pipeline Service to create a
   pipeline run from the static configuration. 
 * **Kubernetes resources**: The Pipeline Service calls the Kubernetes API
-  server to create the necessary Kubernetes resources
-  ([CRDs](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/))
-  to run the pipeline.   
+  server. The API server creates the Kubernetes resources
```
Comment (joeliedtke) [inline content/docs/pipelines/overview/pipelines-overview.md]: I would recommend keeping the original version of the "Kubernetes resources" section. Its a bit shorter, and I think it more correct to say that the API calls to the Kubernetes API create resources required by the pipeline run.

File: content/docs/pipelines/overview/pipelines-overview.md [hunk 2]
```diff
@@ -33,9 +33,10 @@ The following are the goals of Kubeflow Pipelines:
 * Easy re-use: enabling you to re-use components and pipelines to quickly 
   create end-to-end solutions without having to rebuild each time.
 
-In 
-[Kubeflow v0.1.3 and later](https://github.com/kubeflow/pipelines/releases/tag/0.1.3),
-Kubeflow Pipelines is one of the Kubeflow core components. It's automatically deployed during Kubeflow deployment.
+Kubeflow Pipelines is available as a standalone installation and as a core Kubeflow component.
```
Comment (joeliedtke) [inline content/docs/pipelines/overview/pipelines-overview.md]: Suggested revision: Focuses on KFP as a component of Kubeflow, and updates installation options to be a list.

Kubeflow Pipelines is available as a core component of Kubeflow or as a standalone installation.

* Learn more about [installing Kubeflow](/docs/started/getting-started/).
* Learn more about [installing Kubeflow Pipelines standalone](/docs/pipelines/installation/overview/).

**Comment by joeliedtke** [comment]
This is available to preview at:
https://deploy-preview-1791--competent-brattain-de2d6d.netlify.com/docs/pipelines/overview/pipelines-overview/

File: content/docs/pipelines/overview/pipelines-overview.md (hunk 1 above)
Comment (kbhawkey) [reply content/docs/pipelines/overview/pipelines-overview.md]: 👍

File: content/docs/pipelines/overview/pipelines-overview.md (hunk 2 above)
Comment (kbhawkey) [reply content/docs/pipelines/overview/pipelines-overview.md]: 👍

**Comment by joeliedtke** [comment]
/lgtm
/approve

Thanks for working on this @kbhawkey!

**Pull request merged**